core/src/apps/benfen/tx_parser.py

```
@@ -0,0 +1,432 @@
+    owner_bytes = parser.read_bytes(ADDRESS_LENGTH)
+    owner = f"0x{binascii.hexlify(owner_bytes).decode()}"
+    price = parser.read_u64()
+    budget = parser.read_u64()
+    return {"payment": payments, "owner": owner, "price": price, "budget": budget}
+
+
+def parse_transaction_expiration(parser) -> dict | None:
+    expiration_type = parser.read_u8()
+    if expiration_type == 0:
+        return {"type": "None", "value": None}
+    elif expiration_type == 1:
```

Ruff (0.8.2)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)